explorer/src/lib/services/gql-queries.js

```diff
@@ -15,7 +15,6 @@ fragment TransactionInfo on SpentTransaction {
     gasLimit,
     gasPrice,
     id,
-    isDeploy,
```

Comment:
Why are you removing this?
I still see it in the schema and we use it to determine the transaction's `method` property.

Reply:
That's a mistake / leftover from when it looked like the API had changed.